discovery/openstack/instance.go

```diff
@@ -143,7 +144,10 @@ func (i *InstanceDiscovery) refresh() (*targetgroup.Group, error) {
 
 	// OpenStack API reference
 	// https://developer.openstack.org/api-ref/compute/#list-servers
-	opts := servers.ListOpts{}
+	var opts servers.ListOpts
+	opts = servers.ListOpts{
```

Comment:
(nit) it could be only one line as before with the `:=` operator: `opts := servers.ListOpts{...}`

Reply:
Done